bin/rspecq

```diff
@@ -100,14 +107,16 @@ if opts[:report]
     build_id: opts[:build],
     timeout: opts[:report_timeout],
     redis_host: opts[:redis_host],
+    fail_fast: opts[:fail_fast]
   )
 
   reporter.report
 else
   worker = RSpecQ::Worker.new(
     build_id: opts[:build],
     worker_id: opts[:worker],
-    redis_host: opts[:redis_host]
+    redis_host: opts[:redis_host],
+    fail_fast: opts[:fail_fast]
```

Comment:
This should be moved to a separate attribute set via an `attr_accessor`, like `files_or_dirs_to_run`, since it's not a strictly required argument to initialize a worker.

Reply:
966619b